### DISCUSSION TITLE
Is the $data reference a part of jsonschema

### DISCUSSION THREAD

**Discussion by Jeppeotte** [python-jsonschema/jsonschema#1057] [Q&A] [answered]
I have seen that some jsonschema packages like AJV for javascript has the oppotunity to refference the data in the schema byt using $data. Is this possible for in this package? I cant seem to find any documentation about it

**Comment by Julian** [comment, marked as answer]
It's not a part of JSON Schema (the specification).

Though there is certainly at least one [third party JSON Schema vocabulary](https://json-everything.net/json-schema/#schema-data-2022) specifying behavior for such a thing, and you certainly could (or at least I suspect you could) implement support for it on top of this library using just public APIs.
